src/app/order/OrderStatus.tsx

```diff
@@ -16,6 +16,11 @@ const OrderStatus: FunctionComponent<OrderStatusProps> = ({
     supportEmail,
     supportPhoneNumber,
 }) => {
+
+    const validateMandate = useCallback(() => {
```

Comment:
should this function be called `getMandateProvider` ?

Reply:
You are right, it makes more sense.